src/Controller/AwsStorage.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Controller;
+
+use App\Command\CommandDispatcherTrait;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use League\Flysystem\AwsS3v3\AwsS3Adapter;
+use League\Flysystem\Filesystem;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\Form\Extension\Core\Type\FileType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Aws\Credentials\CredentialProvider;
+
+class AwsStorage extends AbstractController
+{
+    use CommandDispatcherTrait;
+
+    /**
+     * @Route("/cfdoc/aws", name="aws_storage_file")
+     */
+    public function awsStorage(Request $request)
+    {
+        $form = $this->createFormBuilder()
+        ->add('upload', FileType::class, array('label' => 'File Upload'))
+        ->add('save', SubmitType::class, array('label' => 'Submit'))
+        ->getForm();
+
+        $form->handleRequest($request);
+        $filesystem = $this->configuration();
+
+        if ($form->isSubmitted() && $form->isValid()) {
+            $data = $form->getData();
+            $fileName = $data['upload']->getClientOriginalName();
+            $file = $data['upload']->getRealPath();
+            if ($filesystem->has($fileName))
+            {echo '<div class="message">Already exists</div>';}
+            else
+            {
+                $stream = fopen($file, 'r+');
+                $result =$filesystem->writeStream($fileName, $stream);
+                fclose($stream);
+                echo '<div class="message">File Uploaded Successfully..!!</div>';
+            }
+        }
+       $aws_data = $filesystem->listContents();
+        return $this->render('framework/doc_tree/aws_upload.html.twig', array(
+            'form' => $form->createView(),
+            'aws_files' => $aws_data,
+        ));
+    }
+
+    private function configuration()
+    {
+        $provider = CredentialProvider::defaultProvider();
+        $client = \Aws\S3\S3Client::factory([
+        'credentials' => $provider,
```

Comment:
The `defaultProvider` is the default, so this line (and lien 56) are not needed if no other credential options will be supported.